imod_coupler/drivers/ribametamod/exchange.py

```diff
@@ -53,8 +54,8 @@ def reset(self) -> None:
             self.realised_negative[flux_label][:] = 0.0
 
     def _check_valid_shortage(self, shortage: NDArray[np.float64]) -> None:
-        eps: float = 1.0e-06
-        if np.any(np.logical_and(self.demand > eps, shortage > eps)):
+        eps: float = 1.0e-05
```

Comment:
One could alternatively apply the check to the realised fraction, rather than the shortage and require that the realised fraction in this particular case be greater than 0.99 or so, because the realised fraction is being derived here